### PR TITLE
Add AttachmentWithReplies method for sending attachment together with replies.

### PR DESCRIPTION
**Pull request by opiuman**
Be able to send attachment with replies in one reply

### CODE REVIEW

File: messenger.go [hunk 1]
```diff
@@ -330,6 +330,15 @@ func (m *Messenger) Send(to Recipient, message string) error {
 	return m.SendWithReplies(to, message, nil)
 }
 
+//SendGeneralMessage will send the GenericTemplate message
```
Comment (paked) [inline messenger.go]: Would you mind putting a space here, between the `//` and `Send...`, want to make sure that we're following good [GoDoc comment styling](https://blog.golang.org/godoc-documenting-go-code).

File: response.go [hunk 2]
```diff
@@ -299,10 +331,11 @@ type SendMessage struct {
 	Message   MessageData `json:"message"`
 }
 
-// MessageData is a text message with optional replies to be sent.
+// MessageData is a text Or attachment message with optional replies to be sent.
```
Comment (paked) [inline response.go]: I'd change this to:

> // MessageData is a message consisting of text or an attachment, with an additional selection of optional quick replies.

**Review by paked — CHANGES_REQUESTED** [review]
Sorry it's taken me so long to get back to you on this!

Thanks for the PR, I've made a few comments about stylistic changes. After that, everything should be good.

File: messenger.go (hunk 1 above)
Comment (opiuman) [reply messenger.go]: Done.

File: response.go (hunk 2 above)
Comment (opiuman) [reply response.go]: make sense. done

**Comment by opiuman** [comment]
The change has made.

**Pull request merged**